### PR TITLE
add confirmation modal and cancel button

### PR DESCRIPTION
**Pull request by Alina-Rosenbusch**
Hey, I implemented a new function into my app, which is pre step for a delete function. Please have a look at this step 😁.

### CODE REVIEW

File: components/HomeArenaList/index.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
-
 import styled from "styled-components";
 import useSWR from "swr";
 import { useState } from "react";
+import { BsTrash3Fill } from "react-icons/bs";
+import {StyledArena, ModalOverlay, Modal, StyledCancelButton} from "./styles";
 
 const BookedArenas = () => {
+  //Delete confirmation
```
Comment (Roland-Hufnagel) [inline components/HomeArenaList/index.js:8]: please remove unnecessary comments

File: components/HomeArenaList/index.js [hunk 2]
```diff
@@ -42,40 +53,23 @@ const BookedArenas = () => {
               {entry.sTime}-{entry.eTime}
             </p>
             <p className="entry_date">{formatDate(entry.date)}</p>
```
Comment (Roland-Hufnagel) [inline components/HomeArenaList/index.js:55]: Ahh! Here you use classnames. We don't want to use Classnames since we are working with styled-components.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Alina, I was really wondering about the DB-Connection in your code. Then I saw that you did US6 before. Okay. 
I did not find a vercel deployment. Haven't you deployed it yet?
Nevertheless I left some comments for you. Please try to adapt your code.

File: components/ArenaList/index.js [hunk 3]
```diff
@@ -23,9 +22,11 @@ const ListArenas = () => {
     <>
       {arenas.map((arena) => {
         return (
-          <StyledArenaList key={arena._id} className="arenaListEntry">
-            <p className="arena">{arena.arenaName}</p>
-          </StyledArenaList>
+          <StyledArenaListWrapper key={arena._id}>
```
Comment (Roland-Hufnagel) [inline components/ArenaList/index.js:25]: While mapping now you create a lot of ``<ul>``s. That is not what you want. You want one ``<ul>``as a wrapper around and then with the map-function create the ``<li>``s. (you can check it in the dev-tools)

File: components/HomeArenaList/index.js [hunk 4]
```diff
@@ -34,48 +56,37 @@ const BookedArenas = () => {
...
-          <StyledArena key={entry._id} className="entry">
-            <h3>{entry.title}</h3>
-            <h4>{entry.bookedArena}</h4>
-            <p>{entry.bookedPerson}</p>
-            <p className="entry_time">
-              {entry.sTime}-{entry.eTime}
-            </p>
-            <p className="entry_date">{formatDate(entry.date)}</p>
-          </StyledArena>
+          <StyledHomeArenaWrapper key={entry._id}>
```
Comment (Roland-Hufnagel) [inline components/HomeArenaList/index.js:59]: Here is the same point as above. You create a lot of ``<ul>``s instead of ``<li>``s.

**Review by Roland-Hufnagel — CHANGES_REQUESTED** [review]
Hi Alina, you did a good refactoring but there is still a bug with your lists. Please fix this and then you got it 👍

**Review by Kheirah — APPROVED** [review]
Looking good!

**Pull request merged**